metadata-models/src/main/pegasus/com/linkedin/metadata/search/SearchEntity.pdl

```diff
@@ -27,4 +27,6 @@ record SearchEntity {
   }] = []
 
   features: optional map[string, double]
+
+  score: optional double
```

Comment:
So if we have this new score, it doesn't mean we won't have features right?

Reply:
This score is the score from elasticsearch. we still have features in this pr for future usage when doing a second pass ranking